src/events/broker.py

```diff
@@ -350,6 +351,14 @@ def _on_storage_attached(self, event: StorageAttachedEvent) -> None:
                 ]
             )
 
+        if self.charm.substrate == "k8s":
+            self.workload.exec(
+                ["chown", "-R", "kafka:root", f"{self.workload.paths.data_path}/{STORAGE}"]
```

Comment:
**nitpick, non-blocking**: If you need to revisit the PR, could we use the literal for the user?

Also **question**: We usually use the same `USER:GROUP` everywhere, is the `root` group necessary?

Reply:
Great point, I'm an idiot, especially after having just commented on your current PR the same thing.
Have updated.